hmftools/orange/do_run_orange_research

```diff
@@ -14,6 +14,18 @@ if [[ -z "${run_dir}" || -z "${output_dir}" ]]; then
     error "Inputs missing. Exiting";
 fi
 
+tumor_sample=$(load_tumor_sample_from_metadata ${run_dir})
+orange_json_path="${run_dir}/orange/${tumor_sample}.orange.json"
+if [[ ${tumor_sample} == "COLO"* ]]; then
```

Comment:
If we rerun ORANGE then we don't need specific handling for COLO since the ORANGE json for COLO should contain the correct DOID already